src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -1632,8 +1632,6 @@ public static void Main()
             System.Console.WriteLine(x1);
             f = false;
         }
-
-        System.Console.WriteLine(x1);
```

Comment:
I don't believe deconstruction declaration can be used in a condition of a loop. Only scenario like that would be interesting.

---
In reply to: [91802162](https://github.com/dotnet/roslyn/pull/15801#discussion_r91802162) [](ancestors = 91802162)